plugins/modules/remote_management/redfish/redfish_command.py

```diff
@@ -582,7 +582,8 @@ def main():
                 result = rf_utils.set_one_time_boot_device(
                     module.params['bootdevice'],
                     module.params['uefi_target'],
-                    module.params['boot_next'])
+                    module.params['boot_next'],
+                    module.params['override_enabled'])
```

Comment:
The new parameter (override_enable) also needs to be added to the AnsibleModule argument_spec dictionary above. See, for example, where uefi_target and boot_next are specified. And to avoid breaking existing playbooks, the new override_enabled param should default to 'Once'. (The default value can be specified in the argument_spec.)

Likewise, add the new param to the DOCUMENTATION string at the beginning of the module so that users will know it is there.

Reply:
Thanks Bill - I knew I had to be missing more to it. I will get this updated!

Reply:
Updated!